conda/recipes/ucx-py/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set version = environ.get('GIT_DESCRIBE_TAG', '0.0.0.dev').lstrip('v') + environ.get('VERSION_SUFFIX', '') %}
+{% set py_version = environ.get('CONDA_PY', '38') %}
+{% set cuda_version = '.'.join(environ.get('CUDA', '11.2').split('.')[:2]) %}
+
+package:
+  name: ucx-py
+  version: {{ version }}
+
+source:
+  git_url: ../../..
+
+build:
+  number: {{ GIT_DESCRIBE_NUMBER }}
+  string: py{{ py_version }}_g{{ GIT_DESCRIBE_HASH }}_{{ GIT_DESCRIBE_NUMBER }}
```

Comment:
Turns out the `g` before `GIT_DESCRIBE_HASH` wasn't needed (as `GIT_DESCRIBE_HASH` already includes a `g`). Dropped in PR ( https://github.com/rapidsai/ucx-py/pull/849 ).